bicep/novnet/main.bicep

```diff
@@ -179,6 +182,9 @@ param storageAccountName string = ''
 @description('Specifies the access tier of the Azure Storage Account resource. The default value is Hot.')
 param storageAccountAccessTier string = 'Hot'
 
+@description('Specifies whether the Azure Storage Account resource allows public access. The default value is enabled.')
+param storageAccountAllowPublicAccess string = 'Enabled'
```

Comment:
@paolosalvatori, wanted to get your thoughts on this. In this PR the default would be to have public access using network security perimeter in learning mode.  Do you think this is a good default, if we disable by default, then users will need to add to the parameters to override?

Reply:
I would keep the ordinary default value storageAccountAllowPublicAccess  = 'Enabled', and set its value to Disabled via parameter. I would also drive the creation / enablement of the network security perimeter  via parameter. Thoughts?

Reply:
@adrianmo @paolosalvatori resolved this.